README.md

```diff
@@ -1,73 +1,83 @@
 # Tarantool
 
-[![Build Status][travis-badge]][travis-url]
-[![Build Status][gitlab-ci-badge]][gitlab-ci-url]
+[![Actions Status][actions-badge]][actions-url]
 [![Code Coverage][coverage-badge]][coverage-url]
 [![Telegram][telegram-badge]][telegram-url]
-[![Slack][slack-badge]][slack-url]
-[![Gitter][gitter-badge]][gitter-url]
-[![Google Groups][groups-badge]][groups-url]
+[![GitHub Discussions][discussions-badge]][discussions-url]
+[![Stack Overflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
+[Tarantool][tarantool-url] is an in-memory database and application server.
```

Comment:
Didn't we rename it into an in-memory computing platform?

Reply:
I would prefer to keep it as is, at least within this PR.

Now the structure of the document is the following:

> Tarantool is an in-memory database and application server.
>
> <...>
>
> Key features of the application server:
>
> <...>
>
> Key features of the database:
>
> <...>

So the application server and the database sections do not require any explanation. If I'll replace 'database and application server' with 'in-memory computing platform', I'll anyway need to clarify that it is an application server and a database. So the most concise way to present key features is to keep existing wording here.

I can return here later and try to find a good wording.

Reply:
I'll do if you or someone else will insist.

Reply:
Well, then you diverge from the naming - and the ideology behind it - at tarantool.io
I can propose the following wording:
```suggestion
[Tarantool][tarantool-url] is an in-memory computing platform consisting of a database and an application server.
```

Reply:
Okay, changed. (I hope I can resolve this discussion so.)